### PR TITLE
No observations in MultiObservedRV

### PR DESCRIPTION
**Pull request by ahartikainen**
Fixes the problem, when MultiObservedRV does not contain observations (DensityDist)

See https://discourse.pymc.io/t/pm-traceplot-error/3524

### CODE REVIEW

**Comment by ahartikainen** [comment]
If anyone can make a simple model to reproduce this error, then we can add it to the tests. Currently the example is too complicated to be included in tests, and I don't know enough of pymc3 to make it simple.

**Comment by ColCarroll** [comment]
I'll try to get to one tonight.

**Comment by junpenglao** [comment]
This should do:
```python
import pymc3 as pm
with pm.Model():
    mu = pm.Normal('mu')
    x = pm.DensityDist('x', pm.Normal.dist(mu, 1.).logp, observed={'value': .1})
    trace = pm.sample()
pm.traceplot(trace)
```

File: arviz/tests/test_data_pymc.py [hunk 1]
```diff
@@ -118,6 +118,19 @@ def test_multiple_observed_rv(self):
         assert not fails
         assert not hasattr(inference_data.sample_stats, "log_likelihood")
 
+    def test_multiple_observed_rv_without_observations(self):
+        with pm.Model():
+            mu = pm.Normal("mu")
+            x = pm.DensityDist(  # pylint disable=unused-variable
```
Comment (OriolAbril) [inline arviz/tests/test_data_pymc.py]: The colon is missing: `pylint: disable=unused-variable`

**Comment by OriolAbril** [comment]
Also, this shows that the lint in azure is not equal to the lint in travis yet, ideas @canyon289?

File: arviz/tests/test_data_pymc.py (hunk 1 above)
Comment (ColCarroll) [inline arviz/tests/test_data_pymc.py]:
```suggestion
            x = pm.DensityDist(  # pylint: disable=unused-variable
```

**Comment by ahartikainen** [comment]
Thanks! LGTM as far as I'm concerned

**Pull request merged**